packages/webview_flutter/webview_flutter/lib/src/navigation_delegate.dart

```diff
@@ -166,4 +174,7 @@ class NavigationDelegate {
 
   /// Invoked when a resource loading error occurred.
   final WebResourceErrorCallback? onWebResourceError;
+
+  /// Invoked when a resource required HTTP authentication.
+  final HttpAuthRequestCallback? onHttpAuthRequest;
```

Comment:
This field can be removed due to https://github.com/flutter/packages/pull/4735#issuecomment-1688957524. But keep it as a constructor parameter though.